{"uuid": "e5a21052-6aae-465c-8620-6132b3c3ecd9", "children": ["e35fef76-08b8-4a3d-bda9-fa038234ecaf"], "befores": [{"name": "login", "status": "broken", "statusDetails": {"message": "Exception: 无该校验方式：no_check\n", "trace": "  File \"/Users/nali/anaconda3/lib/python3.7/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/nali/anaconda3/lib/python3.7/site-packages/_pytest/fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/nali/anaconda3/lib/python3.7/site-packages/_pytest/fixtures.py\", line 792, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"/Users/nali/AutoApiTest/conftest.py\", line 28, in login\n    api_send_check(request.param, case_dict, case_path, relevance)\n  File \"/Users/nali/AutoApiTest/common/unit/api_send_check.py\", line 33, in api_send_check\n    check_result.check_result(test_name, check_data, code, data, case_path, relevance)\n  File \"/Users/nali/AutoApiTest/common/unit/check_result.py\", line 128, in check_result\n    raise Exception(\"无该校验方式：{}\".format(case[\"check_type\"]))\n"}, "steps": [{"name": "POST请求接口", "status": "passed", "attachments": [{"name": "请求接口：", "source": "50eb864d-3288-4e9e-919f-47ac60b6b627-attachment.attach"}, {"name": "用例描述：", "source": "288114fc-e042-47a3-98a9-f57a9d7967c8-attachment.attach"}, {"name": "请求地址", "source": "37bf7824-5e1b-44ff-acdb-cf6f977f002a-attachment.attach"}, {"name": "请求头", "source": "77e41e45-bc5d-485c-b3bd-ba0d443a4149-attachment.attach"}, {"name": "请求参数", "source": "571f7a1e-cc18-4888-8079-133e65a140b3-attachment.attach"}], "start": 1617289727524, "stop": 1617289727526}, {"name": "不校验结果", "status": "passed", "start": 1617289727835, "stop": 1617289727835}], "start": 1617289727521, "stop": 1617289727835}], "start": 1617289727521, "stop": 1617289727937}